portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -34,19 +34,26 @@
 public class CommentsServlet extends HttpServlet {
 
   private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-  
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Task");
```

Comment:
You should be able to use setLimit(maxComments) in the query, then you don't need the counter.

Reply:
I'm confused on the method. Should I use it like query.setLimit(maxComments)? It's not listed as a valid method,

Reply:
Here's an example: https://cloud.google.com/datastore/docs/concepts/queries#cursors_limits_and_offsets

Reply:
As we discussed, this documentation is not the right one, let's keep the counter approach for now